Mage.Client/src/main/java/mage/client/dialog/PreferencesDialog.java

```diff
@@ -301,6 +301,7 @@ public class PreferencesDialog extends javax.swing.JDialog {
     public static final String KEY_CONTROL_YOUR_TURN = "controlYourTurn";
     public static final String KEY_CONTROL_SKIP_STACK = "controlSkipStack";
     public static final String KEY_CONTROL_PRIOR_END = "controlPriorEnd";
+    public static final String KEY_CONTROL_EVERY_END = "controlEveryEnd";
```

Comment:
Also needs to be made much more clear that this new skip would not stop for things entering the stack. I don't think that's communicated clearly enough (or at all) for F11.

Reply:
For F11 - the help text basically says something like 'Skip to End just prior to mine' which I would argue is pretty clear and it's been in use now for at least 5+ years so I guess people know what it means by now if they use it.  Anecdotally a lot of people use it in preference to F9.

Reply:
Neither the buttons nor hint text makes it clear that F5 will stop when something hits the stack and that F11 will not.

Reply:
True, but people will adjust to either using it or ignoring it and using whatever other combination of F5, F9, F11, F2, F3 that they use to skip or not skip during a game.  And skipping until end of turn is actually what the new F1 hotkey will actually do as advertised in its help text.  If it did care about the stack, I agree it should mention that.

As F11 doesn't mention the stack at all in its help text is actually indicative of what it does - it only will stop when the user wanted it to stop, aka the end step just prior to their turn.

Reply:
As a counter-example: all of these skips will still ask players to declare blockers (if they have any creatures available). But the hint makes no mention of that either.

I believe that one or both sets of actions (those that skip past the stack and those that don't) need their information changed to indicate *what* is being skipped.
Whether or not that's within the scope of this PR depends on which skips should have information listed.

Reply:
But there's already a specific option for getting priority if any blockers are available (which the new F1 and the old F11 actually works ok with and will stop if that's selected if there's a blocker already *and* the option's been selected.  They won't stop and give priority if that option's not selected which would have been what the user selected).  There are layers to how it works with a few key checkpoints (one being the stop on any blockers being available etc) which are already working as intended I guess and people to use to whatever level they want.

Reply:
> But there's already a specific option for getting priority if any blockers are available (which the new F1 and the old F11 actually works ok with and will stop if that's selected if there's a blocker already _and_ the option's been selected.
> 
> They won't stop and give priority if that option's not selected which would have been what the user selected.
> 
> There are layers to how it works with a few key checkpoints (one being the stop on any blockers being available which you mentioned) which are already working as intended (I'd argue F5 is problematic as it always stops for triggers when my thoughts on it would be I actually want to skip to the end step, but I think that was intended behavior). People are free to use them to whatever level they want.

You're right about the blockers, didn't realize it was because of a separate setting.